pkg/minishift/registration/redhat.go

```diff
@@ -130,6 +130,14 @@ func (registrator *RedHatRegistrator) Unregister(param *RegistrationParameters)
 			"sudo -E subscription-manager unregister"); err != nil {
 			return err
 		}
+		if _, err := registrator.SSHCommand(
+			"sudo -E subscription-manager remove --all"); err != nil {
```

Comment:
Why we need this as per subscription manager doc, it clearly indicates that `unregister` should strip its subscriptions. 

- https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/quick_registration_for_rhel/un-registering

Reply:
Copying the error from @anjannath 
```
$ sudo -E subscription-manager status
Consumer profile "67845724-d9be-4048-95fa-e81a162c8761" has been deleted from the server. You can use command clean or unregister to remove local profile.
```
So my guess is it is related  to the `local profile` not being cleaned properly